### PR TITLE
Move model init to lightning module.

### PR DESCRIPTION
**Pull request by jaheba**
Issue: #2661

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by jaheba** [comment]
Just checked, the `hparams.yml` looks much better now. Before it was ~200kb, now it's <1kb.

**Comment by lostella** [comment]
cc @shchur for visibility

File: test/torch/model/test_mqf2_modules.py [hunk 1]
```diff
@@ -44,16 +44,19 @@ def test_mqf2_modules(
...
+        {
+            "freq": "1H",
+            "context_length": context_length,
+            "prediction_length": prediction_length,
+            "num_feat_dynamic_real": num_feat_dynamic_real,
+            "num_feat_static_real": num_feat_static_real,
+            "num_feat_static_cat": num_feat_static_cat,
+            "cardinality": cardinality,
+            "distr_output": distr_output,
+        }
```
Comment (lostella) [inline test/torch/model/test_mqf2_modules.py:57]: you don't like the `dict(...)` notation, do you?

**Review by lostella — APPROVED** [review]

File: test/torch/model/test_mqf2_modules.py (hunk 1 above)
Comment (jaheba) [reply test/torch/model/test_mqf2_modules.py:57]: I think `{...}` is visually easier to understand.

**Comment by lostella** [comment]
@jaheba docstrings need to be updated accordingly

**Pull request merged**